table.csv

```diff
@@ -35,6 +35,7 @@ multihashes,,
 sha1,               ,                         0x11
 sha2-256,           ,                         0x12
 sha2-512,           ,                         0x13
+dbl-sha2-256,       ,                         0x56
```

Comment:
I made a comment here with some thoughts on how we can adopt this usages. 

https://github.com/multiformats/multicodec/pull/23#discussion_r86863096

One could argue that a "n times hash function" is a whole new function or we just should have a way to tell the parser that it has to do several rounds.